omim2obo/parsers/omim_entry_parser.py

```diff
@@ -267,18 +261,49 @@ def cleanup_title(
     return label_newcase
 
 
-# todo: explicit_abbrev: Change to List[str]. See: https://github.com/monarch-initiative/omim/issues/129
-def recapitalize_acronyms_in_title(title: str, explicit_abbrev=None, capitalization_threshold=0.75) -> str:
-    """Re-capitalize acronyms / words based on information contained w/in original label"""
-    # todo: probably best to combine explicit abbrevs outside of this func
-    possible_abbreviations = _detect_abbreviations(
-        title, explicit_abbrev, capitalization_threshold=capitalization_threshold)
-    title2 = title
-    for abbrev in possible_abbreviations:
-        title2 = title2.replace(abbrev.upper(), abbrev)
+def recapitalize_acronyms_in_title(title: str, known_abbrevs: Set[str] = None, capitalization_threshold=0.75) -> str:
+    """Re-capitalize acronyms / words based on information contained w/in original label
+
+    todo: If title has been used on cleanup_title() using a replacement_case_method other than the non-default 'lower',
+     then the .replace() operation will not work. To solve, this (a) capture the replacement_case_method used and
+     pass that here, or (b) duplicate the .replace() line and call it on alternative casing variations (.title() and
+     capitalize() (=sentence case)), (c) possibly just compare to word.lower() instead of 'word.
+    todo: (more important): It's probable that .split(' ') is not enough to cover all cases. Should also run the check
+     by splitting on other characters. E.g. consider the following potential cases: "TITLE (ACRONYM)",
+     "TITLE: ACRONYM1&ACRONYM2", "TITLE/ACRONYM" or "TITLE ACRONYM/ACRONYM", "TITLE {ACRONYM1,ACRONYM2}",
+     "TITLE[ACRONYM]",  "TITLE-ACRONYM", or less likely cases such as "TITLE_ACRONYM", "TITLE.ACRONYM". There are quite
```

Comment:
Why do you think these other potential cases exist in the data? Have you seen these in your analysis of the OMIM files?